src/main/java/com/csc366/project/entity/Location.java

```diff
@@ -13,9 +22,85 @@
     @GeneratedValue(strategy=GenerationType.IDENTITY)
     private Long id;
 
-    public Location() { }
+    private LocalDate openDate;
+
+    @ManyToOne(fetch =  FetchType.LAZY)
+    @JoinColumn(name = "address_id")
+    private Address address;
```

Comment:
I feel like this should be `@OneToOne`, since a location probably only has one address